plugins/org.jkiss.dbeaver.ext.altibase/src/org/jkiss/dbeaver/ext/altibase/model/AltibaseDataSource.java

```diff
@@ -555,6 +557,30 @@ public void collectObjectStatistics(DBRProgressMonitor monitor, boolean totalSiz
         }
     }
     
+    ///////////////////////////////////////////////
+    // Altibase Properties
+    public List<AltibaseProperty> getAltibaseProperties(DBRProgressMonitor monitor)
+            throws DBException {
+        return loadPropertyList(monitor);
```

Comment:
`getProperties` maybe?

```suggestion
    @NotNull
    public List<AltibaseProperty> getProperties(@NotNull DBRProgressMonitor monitor) throws DBException {
        return loadPropertyList(monitor);
```

Reply:
Thanks for your comment.

I will change it based on the principle of using common names.